src/CMakeLists.txt

```
@@ -393,7 +393,14 @@ if(ENABLE_OPENCV)
   find_package(OpenCV 4)
   if(NOT OpenCV_FOUND)
     set(ENABLE_OPENCV FALSE CACHE BOOL
-      "Build with OpenCV algorithms (requires Boost, Protobuf 3)" FORCE)
+      "Build with OpenCV algorithms (requires Protobuf 3)" FORCE)
+    # If we have version 4.5.1, all hope is lost
+  elseif(OpenCV_VERSION VERSION_EQUAL 4.5.1)
```

I always forget that technically it's good form to quote these.
```suggestion
  elseif(OpenCV_VERSION VERSION_EQUAL "4.5.1")
```